Source/CarthageKit/Xcode.swift

```diff
@@ -984,10 +1021,10 @@ public func buildScheme(scheme: String, withConfiguration configuration: String,
 					}
 
 			case 2:
-				let firstSDK = sdks[0]
-				let secondSDK = sdks[1]
+				let deviceSDK = sdks[0]
```

Comment:
This still seems fragile. How about a function in `SDK` like:
```swift
func splitSDKs(sdks: [SDK]) -> (simulators: [SDK], devices: [SDK]) {

}
```

Then here we can use that and do split `SDK.simulators.first!`